### ISSUE TITLE
Typing interaction

### ISSUE THREAD

**Issue by biotomas** [Mangatome/WeTookOutStephen#3]
Integrate the typing interaction

**Comment by Mangatome** [comment]
Yes, and we should find a way to make this work with touch. Or to find an alternative for mobile devices?

**Comment by biotomas** [comment]
For mobiles: letters could appear on the screen and the player has to click on them in the correct order or the player draws letters with his finger (this could be complicated to implement).
